quickstart-python/.env.example

```
@@ -0,0 +1,5 @@
+MONDAY_OAUTH_CLIENT_ID=d2example88893415a4783b23e3a4fa4b
+MONDAY_OAUTH_CLIENT_SECRET=d3bd4547example6350b79f09b664c18
+MONDAY_OAUTH_TOKEN_PATH=https://auth.monday.com/oauth2/token
+MONDAY_SIGNING_SECRET=b8bbb24beexamplebbc49d2247e
+MONDAY_OAUTH_BASE_PATH=https://auth.monday.com/oauth2/authorize
```

1. Don't commit
2. End each file with a line break

```suggestion
MONDAY_OAUTH_CLIENT_ID=
MONDAY_OAUTH_CLIENT_SECRET=
MONDAY_OAUTH_TOKEN_PATH=
MONDAY_SIGNING_SECRET=
MONDAY_OAUTH_BASE_PATH=

```

---

All of them are examples, not real values

---

I'd keep `MONDAY_OAUTH_TOKEN_PATH` and `MONDAY_OAUTH_BASE_PATH` and remove the values of the rest, i.e.
```suggestion
MONDAY_OAUTH_TOKEN_PATH=https://auth.monday.com/oauth2/token
MONDAY_OAUTH_BASE_PATH=https://auth.monday.com/oauth2/authorize
MONDAY_OAUTH_CLIENT_ID=
MONDAY_OAUTH_CLIENT_SECRET=
MONDAY_SIGNING_SECRET=

```